src/setOfPeople.js

```diff
@@ -38,7 +38,17 @@
  * @return {Set}
  */
 function setOfPeople(people) {
-  // write code here
+  const visitors = new Set();
+
+  for (const person of people) {
+    if (visitors.has(person)) {
+      visitors.delete(person);
+    } else {
+      visitors.add(person);
+    }
+  }
+
+  return [...visitors];
```

Comment:
You must return `set`

`@return {Set}`
```suggestion
  return visitors;
```